playwright.config.ts

```diff
@@ -0,0 +1,38 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from "@playwright/test";
+
+const config: PlaywrightTestConfig = {
+  testDir: "./tests/e2e",
+  timeout: 30_000,
+  expect: {
+    timeout: 8000,
+  },
+  fullyParallel: true,
+  outputDir: "./tests/artifacts",
+  workers: process.env.CI ? 1 : undefined,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  reporter: "list",
+  globalSetup: "./tests/support/globalSetup",
+  use: {
+    actionTimeout: 0,
+    baseURL: "http://localhost:3000",
+    trace: "retain-on-failure",
+  },
+
+  projects: [
+    {
+      name: "chromium",
```

Comment:
It would be nice to add more browser testing, like firefox and safari, specially safari has some things that would be nice to iron out and make it the browser experience the same in all browsers

Something like this should work

```diff
diff --git a/playwright.config.ts b/playwright.config.ts
index 2ca2f157..bfc17586 100644
--- a/playwright.config.ts
+++ b/playwright.config.ts
@@ -27,6 +27,18 @@ const config: PlaywrightTestConfig = {
         ...devices["Desktop Chrome"],
       },
     },
+    {
+      name: "firefox",
+      use: {
+        ...devices["Desktop Firefox"],
+      },
+    },
+    {
+      name: "webkit",
+      use: {
+        ...devices["Desktop Safari"],
+      },
+    },
   ],

   webServer: {
```

Reply:
I think we might want to do that only on CI, otherwise it takes too long when run locally. f0850694134845180f8b3a0750bb04020cbc9fcc

Another idea would be to run 3 parallel workflows on CI, one for each browser.

TBD: safari tests, they're all broken for some reason.

Reply:
I'll keep this open and play around with running this in parallel and fix the webkit stuff.

Reply:
Okay, will leave the webkit stuff to a follow-up. For now we run the chromium and firefox tests in parallel on CI.
Locally we can specify which browser we would like to target like this:

```
npm run test:e2e -- --project chromium
```

Reply:
f0d7f16d13946b0b3e570f27325560ab1fd32068